src/main/java/org/candlepin/resteasy/filter/SecurityHoleAuthorizationFilter.java

```diff
@@ -132,10 +134,14 @@ private String checkCloudAccountOrganization(AnonymousCloudConsumer anonymousClo
                 anonymousCloudConsumer.getCloudOfferingId());
         }
         catch (OrgForCloudAccountNotCreatedYetException e) {
-            throw new TooManyRequestsException(ORG_DOES_NOT_EXIST_AT_ALL);
+            String msg = "Upstream anonymous owner not yet created by adapters";
```

Comment:
I don't think we should be mentioning the word `adapters` to API responses. Something better may be `Anonymous owner not yet created upstream of candlepin`

Reply:
Yeah, I agree! That message is much better. Updated